### DISCUSSION TITLE
How to allow \ characters to be copied

### DISCUSSION THREAD

**Discussion by Kommynct** [hluk/CopyQ#2916] [Q&A] [answered]
my xwayland clipboard is broken for a specific app, i run

copyq copy "$(xclip -o)" ; copyq clipboard | xclip -i

when ctrl+c is used on the app, essentially, however, this setup gets rid of \, is there any way around that?

**Comment by hluk** [comment, marked as answer]
CopyQ handles **escape sequences** in arguments (for example, `\n` would be translated to a line break), unless you put the arguments after `--`. So the first command would be `copyq copy -- "$(xclip -o)"`.

But a better solution is to **pass data to the standard input** (as with `xclip`):

```
xclip -o | copyq copy -
```

**Reply by Kommynct** [reply]
Thank you so much!
